webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/services/DisplayNode.scala

```diff
@@ -477,32 +480,25 @@ object DisplayNode extends Loggable {
 
   // mimic the content of server_details/ShowNodeDetailsFromNode
   def showNodeDetails(
-      sm:                  FullInventory,
-      nodeAndGlobalMode:   Option[(NodeInfo, GlobalPolicyMode)],
```

Comment:
now, we don't have to wonder if we have the node info - we always have it from node fat. And not getting the `GlobalCompliance` is a normal error, there's no special case to do for it.